common/source/docs/common-viewpro-gimbal.rst

```diff
@@ -69,6 +69,11 @@ Control and Testing
 
 See :ref:`Gimbal / Mount Controls <common-mount-targeting>` and :ref:`Camera Controls <common-camera-controls>` for details on how to control the gimbal using RC, GCS or Auto mode mission commands
 
+IR Rangerfinder Control
```

Comment:
Looks good although I think we could drop the "IR" part 'cuz I guess all laser range finders are IR (?)

Reply:
okay